libs/utils/metrics.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+# TODO: In the future maybe make sense allow the users to configure this parameter
+PROCESSING_COUNT_THRESHOLD = 2
+
+
+def process_distance_violation_for_object(distance_violations):
```

Comment:
It would be good to have a docstring saying what `distance_violations` holds and when this function is called

Reply:
Same for other functions in this file

Reply:
Done